src/components/Card/components/Section/Section.tsx

```diff
@@ -6,28 +6,30 @@ import Subheading from '../../../Subheading';
 import * as styles from '../../Card.scss';
 
 export interface Props {
-  title?: string;
+  title?: React.ReactNode;
   children?: React.ReactNode;
   subdued?: boolean;
   fullWidth?: boolean;
 }
 
 export default function Section({children, title, subdued, fullWidth}: Props) {
-  const headerContent = title ? (
-    <div className={styles.SectionHeader}>
-      <Subheading>{title}</Subheading>
-    </div>
-  ) : null;
-
   const className = classNames(
     styles.Section,
     subdued && styles['Section-subdued'],
     fullWidth && styles['Section-fullWidth'],
   );
 
+  const headerContent =
+    title &&
+    (typeof title === 'string' ? <Subheading>{title}</Subheading> : title);
```

Comment:
This will also need to be wrapped in a `<div className={styles.SectionHeader}>` to preserve spacing as expected.

Reply:
This is wrapped below. However, I will change this as perhaps it's not the most readable / understandable